src/main/webapp/WEB-INF/view/conversations.jsp

```diff
@@ -60,10 +60,11 @@
 
     <h1>Conversations</h1>
 
+	<h4>Public Conversations (Anyone can view):</h4>
     <%
-    List<Conversation> conversations =
```

Comment:
I see here you have removed the "conversations" list but have kept it in the ConversationStore.